core/templates/pages/donate-page/donate-page.component.html

```diff
@@ -1,144 +1,681 @@
-                  frameborder="0"
-                  allowfullscreen
-                  class="position-absolute w-100 h-100 oppia-meet-contributors-iframe">
-          </iframe>
-        </div>
-        <h3 class="oppia-donate-h3">
-          <em>
-            <span [innerHTML]="'I18N_DONATE_PAGE_VIDEO_SUBTITLE' | translate"></span>
-          </em>
-        </h3>
-        <div class="oppia-vision-text-container">
-          <p class="oppia-vision-text">
-            <span [innerHTML]="'I18N_DONATE_PAGE_VISION_TEXT_ONE' | translate"></span>
-          </p>
-          <p class="oppia-vision-text">
-            <span [innerHTML]="'I18N_DONATE_PAGE_VISION_TEXT_TWO' | translate"></span>
-          </p>
+    </div>
+  </div>
+  <div class="oppia-page-card mt-0 mb-0 oppia-donate-page-section-gray-bg">
+    <div class="container">
+      <div class="row">
+        <div class="col-md-6">
+          <div class="row container donate-page-section-heading">
+            <span>
+              {{ 'I18N_DONATE_PAGE_HEADING_1' | translate }}
+              <span class="oppia-donate-page-text-color">
+                {{ 'I18N_DONATE_PAGE_HEADING_2' | translate }}
+              </span>
+            </span>
+          </div>
+          <div class="row">
+            <div class="col-sm-6">
+              <div class="oppia-donate-page-feature-text-container">
+                <div class="oppia-donate-page-text-color oppia-donate-page-feature-text">1,500,000+</div>
+                <div>{{ 'I18N_DONATE_PAGE_STATISTIC_1' | translate }}</div>
+              </div>
+              <div class="oppia-donate-page-feature-text-container">
+                <div class="oppia-donate-page-text-color oppia-donate-page-feature-text">2,000</div>
+                <div>{{ 'I18N_DONATE_PAGE_STATISTIC_2' | translate }}</div>
+              </div>
+            </div>
+            <div class="col-sm-6">
+              <div class="oppia-donate-page-feature-text-container">
+                <div class="oppia-donate-page-text-color oppia-donate-page-feature-text">1</div>
+                <div>{{ 'I18N_DONATE_PAGE_STATISTIC_3' | translate }}</div>
+              </div>
+              <div class="oppia-donate-page-feature-text-container">
+                <div class="oppia-donate-page-text-color oppia-donate-page-feature-text">100+</div>
+                <div>{{ 'I18N_DONATE_PAGE_STATISTIC_4' | translate }}</div>
+              </div>
+            </div>
+          </div>
+          <div class="row mt-5">
+            <div class="col-sm-6">
+              <a class="btn oppia-donate-page-watch-video-btn" href="https://youtu.be/OConyxG7HaM">
+                {{ 'I18N_DONATE_PAGE_WATCH_VIDEO_BUTTON_TEXT' | translate }}
+              </a>
+            </div>
+            <div class="col-sm-6">
+              <a class="btn oppia-donate-page-read-blog-btn" href="https://medium.com/oppia-org">
```

Comment:
```suggestion
              <a class="btn oppia-donate-page-read-blog-btn" 
                 href="https://medium.com/oppia-org">
```

TODO so that this navigates to our own blog later when it is ready?

Reply:
Done.